packages/api/emails/decline/html.pug

```diff
@@ -0,0 +1,28 @@
+div.main
+  img(src=`${process.env.APP_URL}/email/header.jpg` alt='header' style='width:100%; height:auto;')
+  //- image doesn't work when email actually gets send out only works in preview.
+  //- img.ui.mini.rounded.image(src=./header.jpg)
+
+  p Hi #{name},
+
+  p We regret to inform you that your application for #{patienName} were not accepted for a Keep Swimming Foundation grant this quarter. Due to our limited funds, we are unable to provide financial relief to every family who applies. While this is not the result you desired, we do hope you suggest our services to families in need going
+  p forward. As we grow, it is our goal to increase the number of families we help each quarter. If your family has remained in the hospital since you began the prior application process, you may be eligible to apply for our next round of grants if they continue to meet our qualifications. Please Note: The online application must be submitted again and some documents may be requested again.
+
+  p We send our love to your family and wish you all good health.
+
+
+  p Sincerely,
+  p Keep Swimming Foundation
+
+  p Hola,
+
+  p Lamentamos informarle que no fue aceptado para una beca de la Fundación Keep Swimming este trimestre. Debido a nuestros fondos limitados, no podemos proporcionar ayuda financiera a todas las
```

Comment:
Are we going to add the dynamic `name` and `patientName` to the Spanish text here?